pydicom/dataelem.py

```diff
@@ -211,10 +179,13 @@ def value(self, val):
     @property
     def VM(self):
         """Return the value multiplicity (as an int) of the element."""
-        if isMultiValue(self.value):
-            return len(self.value)
-        else:
+        if isinstance(self.value, compat.char_types):
```

Comment:
Actually, does this include PN elements?

Reply:
Nevermind, tested it with both python2.7 and 3.6 and it works OK with multi-value PN elements.